drydock/templates/drydock/k8s/lifecycle/cms.yml

```diff
@@ -28,7 +28,7 @@ spec:
             httpGet:
               httpHeaders:
                 - name: Host
-                  value: {{ LMS_HOST }}
+                  value: {{ CMS_HOST }}
```

Comment:
what about using just `cms` (and `lms`)? They are already in the [`ALLOWED_HOSTS`](https://github.com/overhangio/tutor/blob/46b401690c812448c17b2cd91f90cf765a7a9678/tutor/templates/apps/openedx/settings/lms/production.py#L10) list and this way it reduces additional git diffs when doing changes.